code/development-env/config/ca/README.md

```diff
@@ -1,64 +1,72 @@
-# Create CA root key
+# Development Certificate Authority
+
+## Create CA root key
 
 ```bash
 openssl genrsa -out rootCA.key 2048
 ```
 
-# Create CA root certificate
+## Create CA root certificate
 
 - Give user readable details for root cert; `CN`, `O`, `OU`
 
 ```bash
 openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 1024 -out rootCA.pem
 ```
 
-
-# Create website tls key
+## Create website tls key
 
 ```bash
-openssl genrsa -out datalab.local.key 2048
+openssl genrsa -out datalab.internal.key 2048
```

Comment:
Shouldn't this all be `.localhost`? If we do this then don't we tie ourselves to having to run dnsmasq?